code_snippets/Deploy_Previously_Created_Model.ipynb

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
<p>missing period, and you can link to the console...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1479/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>